src/core/abr/bandwidth_estimator.ts

```diff
@@ -15,7 +15,7 @@
  */
 
 import config from "../../config";
-import EWMA from "./ewma";
+import EWMA from "./utils/ewma";
```

Comment:
I created a `utils` directory to lower the number of files the `src/core/abr`, so it hopefully become more readable.